packages/skeleton-svelte/src/lib/components/Navigation/Navigation.test.ts

```diff
@@ -0,0 +1,204 @@
+	});
+
+	it('Renders in expanded mode', () => {
+		const width = 'width';
+		const widthExpanded = 'widthExpanded';
+		render(Navigation.Rail, {
+			width: width,
+			widthExpanded: widthExpanded,
+			expanded: true
+		});
+		const component = screen.getByTestId(testIds.root);
+		expect(component).not.toHaveClass(width);
+		expect(component).toHaveClass(widthExpanded);
+	});
+
+	for (const prop of ['headerBase', 'headerFlexDirection', 'headerJustify', 'headerItems', 'headerGap', 'headerClasses']) {
+		it(`Correctly applies the header \`${prop}\` prop`, () => {
+			const value = 'bg-green-500';
+			render(Navigation.Rail, {
+				header: mockSnippet('header'),
+				[prop]: value
+			});
+			const component = screen.getByTestId(testIds.header);
+			expect(component).toHaveClass(value);
+		});
+	}
+
+	for (const prop of ['tilesBase', 'tilesFlexDirection', 'tilesJustify', 'tilesItems', 'tilesGap', 'tilesClasses']) {
+		it(`Correctly applies the tiles \`${prop}\` prop`, () => {
+			const value = 'bg-green-500';
+			render(Navigation.Rail, {
+				tiles: mockSnippet('tiles'),
+				[prop]: value
+			});
+			const component = screen.getByTestId(testIds.tiles);
+			expect(component).toHaveClass(value);
+		});
+	}
+
+	for (const prop of ['footerBase', 'footerFlexDirection', 'footerJustify', 'footerItems', 'footerGap', 'footerClasses']) {
+		it(`Correctly applies the tiles \`${prop}\` prop`, () => {
+			const value = 'bg-green-500';
+			render(Navigation.Rail, {
+				footer: mockSnippet('footer'),
+				[prop]: value
+			});
+			const component = screen.getByTestId(testIds.footer);
+			expect(component).toHaveClass(value);
+		});
+	}
+});
+
+describe('NavBar', () => {
+	const testIds = {
+		root: 'nav-bar',
+		tileset: 'nav-bar-tileset'
+	};
+
+	it('Renders the component', () => {
+		render(Navigation.Bar, {});
+		const component = screen.getByTestId(testIds.root);
+		expect(component).toBeInTheDocument();
+	});
+
+	it('Renders all snippets', () => {
+		render(Navigation.Bar, {
+			children: mockSnippet('children')
+		});
+		const component = screen.getByTestId(testIds.root);
+		expect(component).toHaveTextContent('children');
+	});
+
+	for (const prop of ['base', 'background', 'width', 'height', 'padding', 'classes']) {
+		it(`Correctly applies the root \`${prop}\` prop`, () => {
+			const value = 'bg-green-500';
+			render(Navigation.Bar, { [prop]: value });
+			const component = screen.getByTestId(testIds.root);
+			expect(component).toHaveClass(value);
+		});
+	}
+
+	for (const prop of ['tilesBase', 'tilesFlexDirection', 'tilesJustify', 'tilesItems', 'tilesGap', 'tilesClasses']) {
+		it(`Correctly applies the tileset \`${prop}\` prop`, () => {
+			const value = 'bg-green-500';
+			render(Navigation.Bar, { children: mockSnippet('children'), [prop]: value });
+			const component = screen.getByTestId(testIds.tileset);
+			expect(component).toHaveClass(value);
+		});
+	}
+});
+
+describe('NavRail > NavTile', () => {
```

Comment:
Instead of `NavRail > NavTile` you can do this:
```ts
describe('NavRail', () => {
  // The tests for `NavRail`

  describe('NavTile', () => {
  
  });
});
```
Nesting describes will then add those arrows in the console output and group them nicely.

Reply:
I suppose that's true, but I liked all the Tile tests next to one another. Either works for me tbh.

Reply:
Done